src/transformers/__init__.py

```diff
@@ -1530,6 +1546,9 @@
     else:
         from .utils.dummy_tokenizers_objects import *
 
+    if is_torchvision_available():
+        from .models.vit import ViTFeatureExtractor
+
```

Comment:
Nit: please put it at the end to have the same order as for the dict before (will make life easier for automated scripts).